jsk_perception/src/polygon_array_color_likelihood.cpp

```diff
@@ -66,16 +66,16 @@ namespace jsk_perception
       sub_reference_ = pnh_->subscribe(
        "input/reference", 1, &PolygonArrayColorLikelihood::referenceCallback, this);
     }
-    sub_polygon_.subscribe(*pnh_, "input/polygons", max_queue_size_);
-    sub_histogram_.subscribe(*pnh_, "input/histograms", max_queue_size_);
+    sub_polygon_.subscribe(*pnh_, "input/polygons", 10);
+    sub_histogram_.subscribe(*pnh_, "input/histograms", 10);
```

Comment:
is this ok to set 10 at here? should we use max_queue_size at here and introduce new parameter for message filter?